README.md

```diff
@@ -17,6 +17,16 @@ Examples of how to set the PROJ_DIR environment variable:
 * Windows - `C:\...> set PROJ_DIR=C:\OSGeo4W\`
 * Linux/OS X on most shells- `$ export PROJ_DIR=/lib/`
 
+An alternative way to install is with `pip`:
+
+```
+pip install cython
+pip install git+https://github.com/jswhit/pyproj.git
+```
+
+You migh want to run pip with administrative privilegies (e.g. `sudo pip`) or
+perform an user only installation (e.g. `pip install --user`).
```

Comment:
```suggestion
perform a user only installation (e.g. `pip install --user`).
```
I know, I know, [dumb exceptions](https://english.stackexchange.com/a/105117) in human language 😜

Reply:
OMG, didn't know about this. Thanks!